src/app/screens/Enable/index.tsx

```diff
@@ -47,7 +47,9 @@ function Enable(props: Props) {
     await msg.request("addBlocklist", {
       domain: props.origin.domain,
       host: props.origin.host,
-    });
+      // FIXME: typings
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    } as any);
```

Comment:
Instead of comment and `as any` you could use `as FixMe`. Did this in a another project and it worked quite well. We already use this in some places in this codebase as well.

Reply:
Thanks, I have updated the 4 occurrences.